src/schema/metadata/SoftwareVersions.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: SoftwareVersions
+description: |
+  Manufacturer's designation of software version of the equipment that produced
+  the composite instances.
```

Comment:
here again (as with `StationName`) should we rename composite instance? #813

Reply:
Fixed in 90c00d2.